site/using.md

```diff
@@ -7,7 +7,149 @@ All of the features of Flux are accessible from within
 [Weave Cloud](https://cloud.weave.works).
 
 However, `fluxctl` provides an equivalent API that can be used from
-the command line. The `--help` for `fluxctl` is described below.
+the command line.
+
+Download the latest version of the fluxctl client
+[from github](https://github.com/weaveworks/flux/releases).
+
+The `--help` for `fluxctl` is described below.
+
+# Connecting fluxctl to the daemon
+
+You need to tell `fluxctl` where to find the Flux API and to make
+the pod accessible to the command-line client `fluxctl`, you
+will need to expose the API outside the cluster.
+
+A simple way to do this is to piggy-back on `kubectl port-forward`,
+assuming you can access the Kubernetes API:
+
+```sh
+fluxpod=$(kubectl get pod -l name=flux -o name | awk -F / '{ print $2; }')
+kubectl port-forward "$fluxpod" 10080:3030 &
+export FLUX_URL="http://localhost:10080/api/flux"
+```
+
+After that, exporting `FLUX_URL` is enough for `fluxctl` to know how
```

Comment:
This says "After that" but we've already done the action mentioned. I think you can just remove "After that,".